jekyll/_cci2/server-3-install.adoc

```diff
@@ -150,11 +150,11 @@ Or select IAM role and provide:
 * *Role ARN* - 
 https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html[Role ARN for Service Accounts] (Amazon Resource Name) for S3 bucket access.
 
-endif::env-gcp[]
+// Stop hiding from GCP PDF:
```

Comment:
@catherinepope not sure what happened here, is replacing rather than adding the comment a mistake maybe?

Reply:
Hmm, possibly a keyboard spasm! Sorry about that.